packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -148,77 +133,65 @@
   color: var(--_iui-expandable-block-header-label-color, inherit);
 }
 
-/// @arg $type Can be one of: label, sublabel
-@mixin iui-expandable-block-header-label-text($type) {
+.iui-expandable-block-title {
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
   transition: color var(--iui-duration-1) ease;
   @media (forced-colors: active) {
     transition: none;
   }
+}
 
-  @if $type == 'sublabel' {
-    font-size: var(--iui-font-size-0);
-    color: var(--_iui-expandable-block-header-caption-color, var(--iui-color-text-muted));
+.iui-expandable-block-caption {
```

Comment:
some of these styles are the same as above and can be combined, e.g.:

```scss
:is(.iui-expandable-block-title, .iui-expandable-block-caption) {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  transition: color var(--iui-duration-1) ease;
  @media (forced-colors: active) {
    transition: none;
  }
}

.iui-expandable-block-caption {
  font-size: var(--iui-font-size-0);
  color: var(--_iui-expandable-block-header-caption-color, var(--iui-color-text-muted));
}
```

Reply:
still waiting for this

Reply:
updated!